demo/api.md

```diff
@@ -24,6 +24,14 @@
 |---------|------------|------------------------------------------------|
 | [reset](#reset) | `(): void` | Method for a total reset of the radio element. |
 
+## Events
+
+| Event                       | Description                                      |
+|-----------------------------|--------------------------------------------------|
+| `auroFormElement-validated` | Notifies that the `validity` and `errorMessage` values have changed. |
```

Comment:
This description is not necessarily true. It can validate and still show the same message.

```suggestion
| `auroFormElement-validated` | Notifies that the element has been validated.
```